src/Components/Home/Home.css

```diff
@@ -7,17 +7,26 @@
   justify-content: center;
 }
 
-.home-btn {
-  background-color: var(--tertiary-bkgd-color);
-  padding: 1vmin 2vmin 1vmin 2vmin;
-  margin: 1vmin;
-  height: fit-content;
-  border-radius: 5px;
-  cursor: pointer;
-  text-decoration: none;
-  transition: background-color .5s;
+.logo-home {
```

Comment:
I think the ellipse in the logo needs to be bigger here, or the text smaller. Just to be consistent with the relative sizing for the logo on the rest of the pages!